### ISSUE TITLE
Showing errors in console for no reason in vue router project

### ISSUE THREAD

**Issue by OculusMode** [codesandbox/codesandbox-client#2095]
# 🐛 bug report
Error: Property or method "children" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property.

## Description of the problem
I just tried to add a button and tried to console log 'this' or 'this.something' and got into these errors.
I dont see anything wrong with code tho.

### Link to sandbox: [link]() (optional)
https://codesandbox.io/s/vue-routing-example-bx3mg
### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      |
| Browser          | All
| Operating System | All

**Comment by SaraVieira** [comment]
Hey!

Is this still happening? Because your example does not have any errors :(

**Comment by lbogdan** [comment]
Seems to be a `console-feed` issue when `console.log()`-ing a Vue instance:

![image](https://user-images.githubusercontent.com/2083930/61085552-31f8f080-a439-11e9-8674-0cc8e59291d0.png)

(this is the corresponding `shouldTransform()`)
![image](https://user-images.githubusercontent.com/2083930/61085614-5c4aae00-a439-11e9-8288-970b978b567e.png)

Pinging @samdenty .

**Comment by lbogdan** [comment]
@OculusMode This is low priority, as `console.log()`-ing Vue instances is not really a common use-case, and it's more a nuisance than actual breaking behavior.

**Comment by OculusMode** [comment]
Yeah its not currently breaking anything down.
But just a bug when using console.logs.

**Comment by geoidesic** [comment]
It's a huge waste of time if we're getting false errors in the console. I personally think this is pretty important.

**Comment by SaraVieira** [comment]
@OculusMode I cant reproduce anymore, maybe console feed was updated 

@geoidesic Do you have a sandbox with this issue?

**Comment by OculusMode** [comment]
@SaraVieira I just tried to check it and it seems like now it's printing instance correctly.
However, it is giving error while printing something. (`console.log(this)` for example)

`
[Vue warn]: Property or method "children" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. See: https://vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.
`

Should I consider a different issue for that and close because seems like they have resolved it?

**Comment by SaraVieira** [comment]
Yes pls 

That one seems like a Vue warning really, so it comes from the CLI itself

Maybe @lbogdan can help debug that if you share the sandbox

**Comment by OculusMode** [comment]
@SaraVieira yes I can still reproduce it. I just used latest vue sandbox for incase it may have been resolved.
https://codesandbox.io/s/hidden-hill-6516m?file=/src/App.vue

**Comment by geoidesic** [comment]
@SaraVieira here's a sandbox with this issue: https://codesandbox.io/s/working-example-of-mapdynamicmultirowfields-working-v0497

**Comment by y-nk** [comment]
got this bug too. no-one seems to care ?

**Comment by geoidesic** [comment]
Yup, no-one cares apparently, it was auto-marked as stale even though I supplied all the reproducibles required. It's a good way to close bugs... just ignore them and they fix themselves when the issue goes stale >.<

**Comment by andrasbacsai** [comment]
Sorry about that! We are working hard to solve all the issues, but we also need to prioritize them. This one is not the highest priority, as it only causing just a warning. It's not breaking anything.

We always accept contributions from anyone. That will help us a lot!

**Comment by dvago** [comment]
It would be good to fix this, the environment is quite unusable at the moment.

**Comment by tony19** [comment]
@andrasbacsai Since the problem still exists, can this issue be reopened?

**Comment by bennodev19** [comment]
I had the same issue, and it slowed down the code sandbox a lot.
Without @tony19's hint in Stack Overflow, 
I probably would have never figured out that it had something to do with logging the Vue instance.
But once I disabled the logging, it worked as expected..

I agree with @tony19 that the problem still exists (otherwise, it wouldn't have occurred in the sandbox I created yesterday). 
Therefore, it should be reopened, although it has no high priority. 

https://stackoverflow.com/questions/67548658/codesandbox-vue-property-or-method-children-is-not-defined-on-the-instance/67549934#67549934

**Comment by outdoorsman** [comment]
Not sure why this isn't being fixed. I bot closed to this issue which has nothing to do with it being fixed. Could you please consider opening and fixing this? Not good for PR on stackoverflow, see above link.

**Comment by aj-foster** [comment]
Reopening for visibility.

**Comment by joshmakar** [comment]
I added a `console.log()` to a method and started getting this error. Once I removed the `console.log()` the error went away.

**Comment by JamesACS** [comment]
Hello @joshmakar - Have you got an example you can share with us of this?

**Comment by joshmakar** [comment]
Hey @JamesACS. Here's a simple demo. Open the console and then drag one of the names within the list to re-order them. The test method will attempt to console log the event and then the error will be displayed afterwards. If you keep trying to re-order the names, it will eventually lock up.

https://codesandbox.io/p/sandbox/console-log-error-test-x6kn3n
